MultiServer.py

```diff
@@ -777,6 +782,15 @@ def collect_player(ctx: Context, team: int, slot: int):
         register_location_checks(ctx, team, source_player, location_ids, count_activity=False)
         update_checked_locations(ctx, team, source_player)
 
+    if not is_group:
```

Comment:
Is this `not` correct? Seems like it would be checking if `is_group:` no?

Reply:
I don't like that this was never addressed. I could have misinterpreted things but I would have liked to be told that.